db/people.yml

```diff
@@ -464,22 +464,40 @@ cbullard:
   active: true
   aka: []
   avatar: https://github.com/chloeann95.png
-  bio: bio
-  education: []
-  email: cab2344@columbia.edu
+  bio: Chloë grew up in Michigan and attended Michigan State University for her undergraduate degrees in Physics and Mechanical Engineering, where she worked on computational plasma physics research. Afterward she attended the University of Toledo completing a Masters in Chemical Engineering while working on modeling CFD simulations for biomass with NREL. When not working you can find her looking for a new game to play or a new song to learn on the piano.$
+
```

Comment:
no blank line here.  Also, please put in carriage returns every ~80 characters for the bio.

Reply:
done